src/styles/actionmenu.styl

```diff
@@ -0,0 +1,37 @@
+@require 'cozy-ui'
+@require './popover'
+
+.fil-actionmenu-backdrop
+    position   fixed
+    z-index    101
+    top        0
+    right      0
+    bottom     0
+    left       0
+    display    block
+    background rgba(50, 54, 63, 0.5)
+
+.fil-actionmenu
+    @extends $popover
+    position       fixed
+    bottom         em(9px)
+    width          "calc(100% - %s)" % em(16px)
+    margin         0 em(8px)
+    padding-bottom em(5px)
+
+    hr
+        margin-top 0
+
+    a
+        &.fil-action-openwith
```

Comment:
I think we don't need to target the html tag `a` here imho, just `fil-action-openwith` and `fil-action-download` would be enough.